assets/css/_pages.scss

```diff
@@ -1,3 +1,33 @@
 span.highlight {
     background-color: yellow;
 }
+
+.historical-records {
+  table {
+    border-collapse: collapse;
+    margin: $small-spacing 0;
+    table-layout: fixed;
+    width: auto;
+  }
+
+  table:first-child {
+    width: 100%;
+  }
+
+  th {
+    border: 1px solid shade($base-border-color, 25%);
```

Comment:
Properties should be ordered padding, font-weight, color, text-align, background-color, border

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)